core/services/relay/relay.go

```diff
@@ -0,0 +1,27 @@
+package relay
+
+import (
+	"github.com/smartcontractkit/chainlink/core/service"
+	"github.com/smartcontractkit/libocr/offchainreporting2/reportingplugin/median"
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+)
+
+type Relays struct {
+	// TODO: support Ethereum as a relay
+	// Ethereum Relay
+	Solana Relay
+}
+
+type Relay interface {
```

Comment:
nit: classic go style would be naming this Relayer

Reply:
Went with the `Relayer` & `OCR2Provider` as interfaces vs `Relay` & `OCR2Service`. Fits better 👍